contrib/update/update.yml

```diff
@@ -0,0 +1,110 @@
+---
+- name: create new instance
+  hosts: 127.0.0.1
+
+  tasks:
+    - name: create new ssh keys
+      digital_ocean:
+        state: present
+        command: ssh
+        name: serenata-update
+        ssh_pub_key: "{{ lookup('file', '/root/.ssh/id_rsa.pub') }}"
```

Comment:
Here you name an absolute path. It makes your playbook have a dependency outside of the repo.
Since it's a public key, consider adding it to the repo or get it from a server. I personally get it from Github itself.

Although, according to the documentation, I would rather initiate the droplet with an existing key (see flag `ssh_key_ids`).

Reply:
> This sounds good. Just to be sure, you are suggesting something like curl -O https://github.com/<user.>keys, right? In this case, user could be added as an envvar, right?
Yes, but Ansible has a tag that simplifies that. I'll search through my playbooks.

Reply:
> Believe me, I tried (and failed). Would you like to send a PR to this PR implementing that, or pairing on that?

Hum, yeah, let me try it.

Reply:
Fixed by fb58719